src/erlcloud_aws.erl

```diff
@@ -363,6 +370,39 @@ config_env() ->
         _ -> {error, environment_config_unavailable}
     end.
 
+config_with_region(Config) ->
+    case {os:getenv("AWS_REGION"), application:get_env(erlcloud, aws_region)} of
```

Comment:
we do not do aws_access_key_id/aws_secret_access_key/aws_security_token through application env. so may be we should not do it for region either?